modules/widgets/search.php

```diff
@@ -842,22 +841,6 @@ class="widefat"
 				/>
 			</p>
 
-			<!-- Post types control -->
```

Comment:
We need to hide this when instant search is off rather than deleting the code since it still needs to work for Pro version of search. :/

Reply:
This should already work as you described: we split off form rendering for Instant Search enabled widgets in the customizer [here](https://github.com/Automattic/jetpack/commit/cd0a2d6549ef1d83582273f853a51e7d1fdcdcfb#diff-c3a4e65325902f14803f43e29d0891f0R702) in 8.7.

The HTML being removed here is contained exclusively within the `form_for_instant_search` function.

Reply:
Doh!